test/features/caseProgression.feature

```diff
@@ -1,13 +1,26 @@
-@caseProgression
+@functional @caseProgression
 Feature: Set of scenarios to test case creation and case progression
 
   Background:
+    Given the following definition for 'events'
+      | ID                 | Name                    | Description             | Display order | Pre condition sate | Post condition state |
+      | createCase         | Create a case           | Create a case           | 1             |                    | CaseCreated          |
```

Comment:
do we need both Name and Description column? Also Display order perhaps is not much relevant in this case and could be removed?

Reply:
will change